docs/plans/plan-agent-provision-endpoints.md

```
@@ -0,0 +1,49 @@
+# Plan: Service Provisioning Endpoints
+
+## Routes
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)